SwissKnife/calendar/WeekUtils.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+
+def get_week_format_from_timestamp(timestamp_millis: int) -> str:
+    """Get week format from timestamp in milliseconds.
```

Comment:
I would add in the docstring that this function makes use of [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601)